base/bucket_gocb.go

```diff
@@ -104,7 +105,18 @@ func GetCouchbaseBucketGoCB(spec BucketSpec) (bucket *CouchbaseBucketGoCB, err e
 		EnableGoCBLogging()
 	}
 
-	cluster, err := gocb.Connect(spec.Server)
+	connSpec, err := gocbconnstr.Parse(spec.Server)
```

Comment:
The parse/string approach is a nice way to avoid having to build the server string yourself.  +1